core/src/main/java/org/geysermc/geyser/network/GameProtocol.java

```diff
@@ -63,19 +60,12 @@ public final class GameProtocol {
     private static final PacketCodec DEFAULT_JAVA_CODEC = MinecraftCodec.CODEC;
 
     static {
-        SUPPORTED_BEDROCK_CODECS.add(Bedrock_v589.CODEC.toBuilder()
-            .minecraftVersion("1.20.0/1.20.1")
-            .build());
-        SUPPORTED_BEDROCK_CODECS.add(Bedrock_v594.CODEC.toBuilder()
-            .minecraftVersion("1.20.10/1.20.15")
-            .build());
-        SUPPORTED_BEDROCK_CODECS.add(Bedrock_v618.CODEC.toBuilder()
-            .minecraftVersion("1.20.30/1.20.32")
-            .build());
         SUPPORTED_BEDROCK_CODECS.add(Bedrock_v622.CODEC.toBuilder()
             .minecraftVersion("1.20.40/1.20.41")
             .build());
-        SUPPORTED_BEDROCK_CODECS.add(DEFAULT_BEDROCK_CODEC);
+        SUPPORTED_BEDROCK_CODECS.add(DEFAULT_BEDROCK_CODEC.toBuilder()
+            .minecraftVersion("1.20.50/1.20.51")
```

Comment:
Wasn't there something about not having slashes in the latest supported version, or am I misremembering something

Reply:
That was for pinging stuff right? I think that issue might've been just in regards to DEFAULT_BEDROCK_CODEC. Whereas this just a list that is used to create a message of what is supported

Reply:
Ah, excellent. Thanks for double-checking :)